docs/plugins/google-cloud.md

```diff
@@ -134,22 +189,237 @@ This field specifies the timeout for the metrics export in milliseconds.
+| path          | The path of execution from the feature root to this action. eg. '/myFeature/parentAction/thisAction' |
+| status        | 'success' or 'failure' depending on whether or not the action succeeded                              |
+| error         | Only set when `status=failure`. Contains the error type that caused the failure                      |
+| source        | The Genkit source language. Eg. 'ts'                                                                 |
+| sourceVersion | The Genkit framework version                                                                         |
+
+### Generate metrics
+
+These are special action metrics relating to actions that interact with a model.
+In addition to requests and latency, input and output are also tracked, with
+model specific dimensions that make debugging and configuration tuning easier.
+
+| Name                                 | Type      | Description                                |
+| ------------------------------------ | --------- | ------------------------------------------ |
+| genkit/ai/generate/requests          | Counter   | Number of times this model has been called |
+| genkit/ai/generate/latency           | Histogram | Execution latency in ms                    |
+| genkit/ai/generate/input/tokens      | Counter   | Input tokens                               |
+| genkit/ai/generate/output/tokens     | Counter   | Output tokens                              |
+| genkit/ai/generate/input/characters  | Counter   | Input characters                           |
+| genkit/ai/generate/output/characters | Counter   | Output characters                          |
+| genkit/ai/generate/input/images      | Counter   | Input images                               |
+| genkit/ai/generate/output/images     | Counter   | Output images                              |
+| genkit/ai/generate/input/audio       | Counter   | Input audio files                          |
+| genkit/ai/generate/output/audio      | Counter   | Output audio files                         |
+
+Each generate metric contains the following dimensions:
+
+| Name            | Description                                                                                          |
+| --------------- | ---------------------------------------------------------------------------------------------------- |
+| modelName       | The name of the model                                                                                |
+| featureName     | The name of the parent feature being executed                                                        |
+| path            | The path of execution from the feature root to this action. eg. '/myFeature/parentAction/thisAction' |
+| temperature     | The temperature parameter passed to the model                                                        |
+| maxOutputTokens | The maxOutputTokens parameter passed to the model                                                    |
```

Comment:
tokens, topk, topp, temperature removed

Reply:
that high cardinality gets ya every time 😄  thanks!